core/src/smartcontracts/isi/expression.rs

```diff
@@ -523,22 +523,30 @@ mod tests {
         let manager_signatory_set = Value::Vec(vec![manager_signatory.clone()]);
         let condition = IfBuilder::condition(And::new(
             Greater::new(
-                EvaluatesTo::new_unchecked(ContextValue::new("usd_quantity").into()),
+                EvaluatesTo::new_unchecked(
+                    ContextValue::new(Name::from_str("usd_quantity").unwrap()).into(),
```

Comment:
Maybe use `expect` as you do in `core/src/tx.rs`? Relevant for other places too.

Reply:
Yea I will change it.